docs/resources/fabric_service_token.md

```diff
@@ -41,6 +41,35 @@ resource "equinix_fabric_service_token" "test" {
 }
 ```
 
+Aside Port Service Token
+```terraform
+resource "equinix_fabric_service_token" "test"{
+  type = "VC_TOKEN"
+  description = "Zside COLO Service Token"
```

Comment:
This corresponds to Zside. The attribute z_side is given in the body of this resource too. Is this the correct implementation?

Reply:
Title of this example was wrong